code/modules/mob/living/simple_animal/bot/secbot.dm

```diff
@@ -195,7 +195,7 @@
 	..()
 	if(istype(W, /obj/item/weldingtool) && user.a_intent != INTENT_HARM) // Any intent but harm will heal, so we shouldn't get angry.
 		return
-	if(W.force && !target && W.damtype != STAMINA)
+	if(!isscrewdriver(W) && !locked && (W.force) && (!target) && (W.damtype != STAMINA))//If the target is locked, they are recieving damage from the screwdriver
```

Comment:
You have old stuff. This reintroduces a bug

Reply:
Yeah I definitely downloaded old files.. I'll revert that in an hour or two!

Reply:
It is reverted. I'll check all the files tomorrow for hidden old code.